src/css/components/Nav.scss

```diff
@@ -0,0 +1,71 @@
+
+.Nav {
+  background-color: color-neutral(1);
+  display: flex;
+  flex-direction: column;
+
+  a, .nav-item {
+    color: color-neutral(9);
+  }
+
+  .Icon {
+    height: 1.2em;
+    width: 1.2em;
+  }
+
+  &.is-collapsed {
+    max-width: 64px;
+    .nav-item-title {
+      display: none !important;
+    }
+
+    .nav-header {
+      padding: 16px;
+    }
+
+    .nav-collapse {
+      .Icon {
+        transform: rotate(180deg);
+      }
+    }
+  }
+
+  .nav-header {
+    border-bottom: 1px solid color-neutral(5);
+    padding: 16px;
+    font-size: 1.4rem;
+  }
+
+  .nav-collapse {
+    border-top: 1px solid color-neutral(5);
+    cursor: pointer;
+    padding: 12px 24px;
+    .Icon {
+      transition: transform 0.2s ease-in-out;
+    }
+  }
+
+  li {
+    background-color: color-neutral(1);
+    border-radius: 5px;
+    cursor: pointer;
+    height: 40px;
+    list-style: none;
+    margin: 12px;
+    padding: 12px;
+    user-select: none;
+
+    &:hover {
+      background-color: color-neutral(3);
+    }
+
+    &:active {
+      background-color: color-neutral(4);
+    }
+
+    &.is-active {
+      background-color: color-primary(6);
+      color:color-neutral(1);
+    }
+  }
+}
```

Comment:
Could the class structure could be cleaned up a little so the output is more like:
```
<div class="Nav">
  <div class="Nav-header">
    <div class="Nav-headerIcon" /> ...etc
  <div class="Nav-body">
    <div class="Nav-item">
    <div class="Nav-item">
  <div class="Nav-footer">
</div>
```

Reply:
I think I've cleaned it up so that its more aligned with what you're asking, but I definitely would appreciate a second pass on reviewing it again.